packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -159,6 +161,38 @@ class ConversationService {
     };
   }
 
+  private async sendConfirmationEphemeral(
+    conversationId: string,
+    payloadBundle: PayloadBundleOutgoingUnsent
+  ): Promise<PayloadBundleOutgoing> {
+    const confirmationContent = payloadBundle.content as ConfirmationContent;
+
+    const confirmationObject: PayloadBundleOutgoingUnsent = {
+      ...payloadBundle,
+      type: PayloadBundleType.CONFIRMATION,
+    };
+
+    await this.send(conversationId, confirmationObject);
```

Comment:
Here the method sends the `confirmation` and the `ephemeralConfirmation` at the same time.
The timeline should be this way:
- send `confirmation` right away ;
- wait until timer is out ;
- send `ephemeralConfirmation`.

Also, if you use `timeout` for that, you are going to run into issues if you have timers > a few days (don't remember exactly the number, something like 20)